test/CMakeLists.txt

```diff
@@ -1,199 +1,152 @@
-
-#--------------------------------------------------
-
-configure_file( ${DD4hep_ROOT}/cmake/run_test_package.sh run_test_${PackageName}.sh @ONLY)
-INSTALL(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/run_test_${PackageName}.sh
-  DESTINATION bin )
-
-macro(SET_TEST_ENV test_name)
+function(SET_TEST_ENV test_name)
   # Make sure to point our current installation for the compact files
-  SET_TESTS_PROPERTIES( t_${test_name}
+  SET_TESTS_PROPERTIES( ${test_name}
     PROPERTIES
       FAIL_REGULAR_EXPRESSION  "Exception;EXCEPTION;ERROR;Error"
       ENVIRONMENT
         K4GEO=${CMAKE_INSTALL_PREFIX}/share/k4geo
```

Comment:
This points to the install location, so it should probably be something else?
Also LD_LIBRARY_PATH is no longer being set?

Reply:
I removed `K4GEO`, which seems not to be needed. 
`LD_LIBRARY_PATH` is set at the bottom for all tests. Maybe I can do the same for `FAIL_REGULAR_EXPRESSION` and remove the function